src/meta/src/hummock/compactor_manager.rs

```diff
@@ -212,6 +219,14 @@ impl CompactorManager {
         tracing::info!("Removed compactor session {}", context_id);
     }
 
+    pub fn set_compactor_config(
+        &self,
+        context_id: HummockContextId,
+        config: CompactorRuntimeConfig,
+    ) {
+        self.policy.read().set_compactor_config(context_id, config);
```

Comment:
The logic of setting config in both policies seems the same. Can we change it to below?

```rust
if let Some(compactor) = self.policy.read().get_compactor(context_id) {
    compactor.set_config(config);
}
```